app/models/concerns/turbo/broadcastable.rb

```diff
@@ -297,7 +311,10 @@ def broadcast_rendering_with_defaults(options)
         # Add the current instance into the locals with the element name (which is the un-namespaced name)
         # as the key. This parallels how the ActionView::ObjectRenderer would create a local variable.
         o[:locals] = (o[:locals] || {}).reverse_merge!(model_name.element.to_sym => self)
-        o[:partial] ||= to_partial_path
+        # if the html or inline options are passed it will skip setting a partial from #to_partial_path
+        unless o.include?(:html) || o.include?(:inline)
```

Comment:
Don't like having both html and inline. Let's just go with `html`.

Reply:
Okay sounds good to me I will update it

Reply:
@dhh all done